### PR TITLE
Fixed renderer stats ['H' panel] cause by post-processing.

### PR DESCRIPTION
**Pull request by ohmed**

### CODE REVIEW

File: renderer.js [hunk 1]
```diff
@@ -44,6 +44,23 @@ function bindCanvas(c) {
   // renderer.gammaFactor = 2.2;
   renderer.shadowMap.enabled = true;
   renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+
+  // reload renderer WebGLInfo.reset function
```
Comment (avaer) [inline renderer.js:48]: What is being reloaded?

File: renderer.js [hunk 2]
```diff
@@ -44,6 +44,23 @@ function bindCanvas(c) {
...
+  // in order to continue chaning frames value
```
Comment (avaer) [inline renderer.js:49]: What is a chaining frames value?

File: renderer.js [hunk 3]
```diff
@@ -44,6 +44,23 @@ function bindCanvas(c) {
...
+  // in other case [.autoReset = false] it'll broke Avatars hair rendering [not sure why]
```
Comment (avaer) [inline renderer.js:50]: This is worrying. Why would the stats affect the rendering? This should be debugged.

**Review by avaer — CHANGES_REQUESTED** [review]
Looks like more debugging needed, because the code is mysterious and relying on unexplained comments.

File: renderer.js (hunk 1 above)
Comment (ohmed) [reply renderer.js:48]: Default renderer.info.reset method

File: renderer.js (hunk 2 above)
Comment (ohmed) [reply renderer.js:49]: Changing renderer.info.frames number.
Is increased each .info.reset() call.
If not called automatically [renderer.info.autoReset = false] it'll result in renderer.info.frames not updating and will cause issues.

**Comment by ohmed** [comment]
I dig a bit deeper you can get to this
image
frame is used to update skinned meshes [if frame the same, update is skipped].
and it's probably related to PostProcess or smth.
Should I continue or just contact somebody who did this part of code [rendering head etc] ?

File: renderer.js (hunk 1 above)
Comment (avaer) [reply renderer.js:48]: That's not really "reload". Reloading the page to collect stats is something very different, so this comment is confusing.

File: renderer.js (hunk 2 above)
Comment (avaer) [reply renderer.js:49]: Where is the chain?

**Comment by avaer** [comment]
☝️ 
Links are more useful than screenshots. I found it by searching myself, but that's pure luck: https://github.com/mrdoob/three.js/blob/fadb6ab1cfcf917e8287e2cca6292aae6ef15793/src/renderers/WebGLRenderer.js

**Comment by avaer** [comment]
I don't think this is the right direction, since the research was not done and this code causes mysterious problems.

We can re-open when we have a better understanding of the code and what to fix.

File: renderer.js (hunk 2 above)
Comment (avaer) [reply renderer.js:49]: > If not called automatically [renderer.info.autoReset = false] it'll result in renderer.info.frames not updating and will cause issues.

What issues?

**Pull request closed without merging**